frontend/screens/Map.js

```diff
@@ -405,8 +430,8 @@ const Map = () => {
           setRegion({
             latitude: data.geometry.location.lat,
             longitude: data.geometry.location.lng,
-            latitudeDelta: region.latitudeDelta,
-            longitudeDelta: region.longitudeDelta,
+            latitudeDelta: 0.0922,
```

Comment:
why is this numbered?

Reply:
onPoiClick does not gives us the delta values
and the delta values are the ones that help the map to zoom
Therefore fixing the issue of when we click on a location the map will automatically zoom into it